io.github.rafaelfassi.QLogExplorer.yaml

```diff
@@ -0,0 +1,33 @@
+app-id: io.github.rafaelfassi.QLogExplorer
+runtime: org.kde.Platform
+runtime-version: '5.15'
```

Comment:
```suggestion
runtime-version: '5.15-21.08'
```

Reply:
Thanks for the suggestion.
It's applied now.